src/typeform-types.ts

```diff
@@ -71,6 +71,7 @@ export namespace Typeform {
           id: string
           title: string
           last_updated_at: string
+          created_at: string
```

Comment:
Is this field new? If so, could you please also update the documentation?

Reply:
The field is already present in docs on dev portal: https://www.typeform.com/developers/create/reference/retrieve-forms/